app/src/main/kotlin/de/digitalService/useID/ui/components/pin/PinDigitField.kt

```diff
@@ -22,10 +21,17 @@ import androidx.compose.ui.unit.dp
 import de.digitalService.useID.ui.theme.UseIdTheme
 
 @Composable
-fun PinDigitField(input: Char?, obfuscation: Boolean, placeholder: Boolean, modifier: Modifier = Modifier) {
+fun PinDigitField(
+    input: Char?,
+    obfuscation: Boolean,
+    placeholder: Boolean,
+    modifier: Modifier = Modifier
+) {
     Box(
         modifier = modifier
-            .size(30.dp)
+            // To prevent the box from resizing after a first digit is entered, calulcate a suitable height
+            // with respect to the current system font scaling
+            .defaultMinSize(minWidth = 30.dp, minHeight = (15 + (LocalDensity.current.fontScale * 30)).dp)
```

Comment:
So we do look up the font scaling for adjusting the UI after all? 🙃

Reply:
Yes, I am not sure if this is a viable solution code wise, but I tried many things and wasn't able to find a way to make it work without taking the current scaling into account. Let me no if this is a no go and I will look into it again to find a different way :)

Reply:
No, it's fine. Doesn't hurt here. 👍